res/css/views/context_menus/_IconizedContextMenu.scss

```diff
@@ -50,21 +50,21 @@ limitations under the License.
         }
 
         // round the top corners of the top button for the hover effect to be bounded
-        &:first-child .mx_AccessibleButton:first-child {
+        &:first-child .mx_AccessibleButton:not(.mx_AccessibleButton_hasKind):first-child {
```

Comment:
These changes stop the styles from clobbering nested components.

Before | After
--- | ---
<img width="406" alt="Screen Shot 2022-01-14 at 10 27 03 PM" src="https://user-images.githubusercontent.com/558581/149608822-459ad218-a4df-43bf-9633-61d9df6d6289.png"> | <img width="414" alt="Screen Shot 2022-01-14 at 10 27 52 PM" src="https://user-images.githubusercontent.com/558581/149608823-7f7642ab-f744-4a2d-b767-764af43b1809.png">

Reply:
The height & border-radius difference between the date field & go button feel inconsistent

Reply:
Updated 👍

<img width="426" alt="" src="https://user-images.githubusercontent.com/558581/150060219-1bc88d65-4125-40f0-bceb-1ff2e207753b.png">


Reference: there is some precedent for adjusting the border radius next to a button in the Settings -> Notifications -> Keywords

<img width="867" alt="" src="https://user-images.githubusercontent.com/558581/150056781-94408418-60ce-414e-a4f8-7c47f9471a0b.png">